case/src/main/kotlin/com/ritense/case/domain/CaseListColumn.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.case.domain
+
+import org.hibernate.annotations.Type
+import javax.persistence.*
+
+@Entity
+@Table(name = "case_list_column")
+data class CaseListColumn(
+    @Id
+    @Column(name = "case_definition_name")
+    val caseDefinitionName: String,
+    @Column(name = "title")
+    val title: String?,
+    @Column(name = "column_key")
+    val key : String,
```

Comment:
Shouldn't this have @,Id

Reply:
Usually we do it with an @EmbeddedId on an object:
https://github.com/valtimo-platform/valtimo-backend-libraries/blob/de6abaec943b131baa39f815f3fea3ad50733d9e/plugin/src/main/kotlin/com/ritense/plugin/domain/PluginActionDefinition.kt#L32

Reply:
changed